cep-XXXX.md

```diff
@@ -0,0 +1,241 @@
+- The individual version, build string, and label for a conda package can be extracted from the OCI `<tag>` by splitting on `-`.
+- The `main` label, which is the default label for conda packages in a channel, is not explicitly listed in the OCI `<tag>` in order to reduce the length of the OCI `<tag>` and to reduce clutter.
+
+## Backwards Compatibility
+
+This specification is not fully backwards compatible with the original `v0` proof-of-concept implementation/specification of conda packages in an OCI registry in the [conda-oci-mirror](https://github.com/channel-mirrors/conda-oci-mirror) project. See the Alternatives section below. The main differences are the construction of the OCI artifact `<name>:<tag>` from the conda package information and the addition of OCI Annotations to the manifest. However, the OCI blob structure is unchanged, so cheap conversion may be possible by uploading only new OCI manifests with the new OCI artifact `<name>:<tag>` and the required OCI Annotations.
+
+The conda channel, subdir, label, and package name regexes are backwards compatible with the current conda implementation and all existing packages on the `defaults` and `conda-forge` channels, except the `__anaconda_core_depends` package on the `defaults` channel. As of 2025-03-10 the label `NOLABEL` is not in use anywhere on anaconda.org. The regex for labels above was pulled from an anaconda.org error message describing the set of valid labels. Finally, anaconda.org usernames/channel names are case insensitive and cannot begin with an underscore.
+
+## Alternatives
+
+- An initial version of this specification was proposed in [CEP PR #70](https://github.com/conda/ceps/pull/70) and received substantial feedback from the community. The current version of the specification is a result of that feedback. Particularly notable feedback included the suggestion of [symbol name mangling techniques and the first version of the shortened encoding used above](https://github.com/conda/ceps/pull/70/files#discussion_r1740438239) by [Bas Zalmstra](https://github.com/baszalmstra).
+
+- The [channel-mirrors project](https://github.com/channel-mirrors) with its associated package [conda-oci-mirror](https://github.com/channel-mirrors/conda-oci-mirror) developed a proof-of-concept implementation of conda packages and repodata stored in an OCI registry. This specification is based on the experience gained from that project, but is not fully backwards compatible with the original implementation. See Backwards Compatibility above. This original implementation and implied specification is referred to as `v0`.
+
+## Sample Implementation
+
+### conda-oci
+
+The [conda-oci](https://github.com/conda-incubator/conda-oci) repository has reference implementations of the procedures defined above for computing the OCI artifact `<name>:<tag>` and for converting a conda artifact into an OCI artifact with a manifest and layers/blobs.
```

Comment:
This is not ready yet, right?

Reply:
Nope. My plan is to have it ready before the vote happens. The code parsing/validation is done, but I need to add in the package structure, CI, and code to convert artifacts.